src/rtp/rtp_callback.c

```diff
@@ -233,7 +233,6 @@ void rtp_recv_callback(struct rtp *session, rtp_event * e)
                 process_sdes(participants, e->ssrc, (rtcp_sdes_item *) e->data);
                 break;
         case RX_APP:
-                pckt_app = (rtcp_app *) e->data;
```

Comment:
This line is duplicit with the same line above, causing clang-tidy to complain about dead variables. Resolution: removed extra line.